web/src/components/feature/PhaseTable.tsx

```diff
@@ -1,36 +1,20 @@
-import { DynamicTable, TDynamicTableInput } from '../common/DynamicTable'
+import { DynamicTable } from '../common/DynamicTable'
 import { MultiflashResponse } from '../../api/generated'
 
-function transformRawResponse(
+function getRows(
   multiFlashResponse: MultiflashResponse,
   cubicFeedFlow: number
-): TDynamicTableInput {
-  const tableInput: TDynamicTableInput = {
-    Phases: [],
-    [`Mass Concentration`]: [],
-    [`Mole Concentration`]: [],
-    [`Mercury Flow (g/d)`]: [],
-  }
-  const phasesInResult: string[] = Object.keys(multiFlashResponse.phaseValues)
-  phasesInResult.forEach((phase, index) => {
-    tableInput['Phases'].push(phase)
-    tableInput['Mass Concentration'].push(
-      multiFlashResponse.phaseValues[phase]['mercury'].toString()
-    )
-    tableInput['Mole Concentration'].push(
-      // mercury has component id 5
-      multiFlashResponse.componentFractions['5'][index].toString()
-    )
-    // TODO: add support for feed in kg/d
-    const hgPhaseFlow =
-      cubicFeedFlow *
-      42.29256 *
-      multiFlashResponse.phaseValues[phase]['percentage'] *
-      multiFlashResponse.componentFractions['5'][index] *
-      200.59
-    tableInput['Mercury Flow (g/d)'].push(hgPhaseFlow.toString())
-  })
-  return tableInput
+): string[][] {
+  const phPhaseFlowFactor = cubicFeedFlow * 42.29256 * 200.59
+  const mercury = multiFlashResponse.componentFractions['5']
```

Comment:
Mistet noen kommentarer her

Reply:
Good point, added issue on fixing the problem indicated in the second comment. First comment is not needed due to a more descriptive variable name, though.